lib/jbuilder/jbuilder_template.rb

```diff
@@ -223,7 +225,7 @@ def _partial_options?(options)
   end
 
   def _is_active_model?(object)
-    object.class.respond_to?(:model_name) && object.respond_to?(:to_partial_path)
+    object.respond_to?(:to_partial_path) && object.class.respond_to?(:model_name)
```

Comment:
Swapped these conditions because `object.respond_to?` is faster than `object.class.respond_to?`, so we can save on the more expensive one when `object.respond_to?(:to_partial_path)` is `false`.